keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -1475,225 +1189,368 @@ prevented. Deleting the entire cluster also leaves resources allocated outside
 of the cluster in an allocated state.
 
 ```go
-// ResourceClaimSpec defines how a resource is to be allocated.
 type ResourceClaimSpec struct {
-    // ResourceClassName references the driver and additional parameters
-    // via the name of a ResourceClass that was created as part of the
-    // driver deployment.
+    // These constraints must be satisfied by the set of devices that get
+    // allocated for the claim.
+    Constraints []Constraint
+
+    // Requests are individual requests for separate resources for the claim.
+    // An empty list is valid and means that the claim can always be allocated
+    // without needing anything. A class can be referenced to use the default
+    // requests from that class.
+    //
+    // +listType=atomic
+    Requests []Request
```

Comment:
can we move Requests above Constraints? Requests (or Devices if we go that way) are the main thing; Constraints and Config are refinements on them.

Reply:
+1

Reply:
Done, and also shuffled around the other structs accordingly (makes the diff larger, but the end result more useful).